TheWarWithin/Priorities/RogueOutlaw.simc

```diff
@@ -13,7 +13,7 @@ actions+=/kick
 # Variables that define the reroll rules for Roll the Bones  Default rule: reroll if the only buff that will be rolled away is Buried Treasure, or Grand Melee in single target without upcoming adds
 actions+=/variable,name=rtb_reroll,value=rtb_buffs.will_lose=(rtb_buffs.will_lose.buried_treasure+rtb_buffs.will_lose.grand_melee&spell_targets.blade_flurry<2)
 # If Loaded Dice is talented, then keep any 1 buff from Roll the Bones but roll it into 2 buffs when Loaded Dice is active. Also reroll 2 buffs with loaded dice up if broadside, ruthless precision and true bearing are all missing and loaded dice is up
-actions+=/variable,name=rtb_reroll,if=talent.loaded_dice,value=(rtb_buffs.will_lose<=buff.loaded_dice.up)|buff.loaded_dice.up&rtb_buffs.will_lose<3&(!rtb_buffs.will_lose.broadside|buff.broadside.remains<11)&(!rtb_buffs.will_lose.ruthless_precision|buff.ruthless_precision.remains<11)&(!rtb_buffs.will_lose.true_bearing|buff.true_bearing.remains<11)
+actions+=/variable,name=rtb_reroll,if=talent.loaded_dice,value=((rtb_buffs.will_lose<=1&buff.loaded_dice.up)|(rtb_buffs.will_lose<1&!buff.loaded_dice.up))|buff.loaded_dice.up&rtb_buffs.will_lose<3&(!rtb_buffs.will_lose.broadside|buff.broadside.remains<11)&(!rtb_buffs.will_lose.ruthless_precision|buff.ruthless_precision.remains<11)&(!rtb_buffs.will_lose.true_bearing|buff.true_bearing.remains<11)
```

Comment:
For expediency, I would merge:

> actions+=/variable,name=rtb_reroll,if=talent.loaded_dice,value=`(rtb_buffs.will_lose<=1*buff.loaded_dice.up)`|buff.loaded_dice.up&rtb_buffs.will_lose<3&(!rtb_buffs.will_lose.broadside|buff.broadside.remains<11)&(!rtb_buffs.will_lose.ruthless_precision|buff.ruthless_precision.remains<11)&(!rtb_buffs.will_lose.true_bearing|buff.true_bearing.remains<11)

This converts internally to `( rtb_buffs_will_lose <= 1 * safenum( buff.loaded_dice.up ) )`, where `safenum` converts a boolean ( `false`/`nil` -> `0`; `true` -> `1` ).

Reply:
Awesome, will get a new pack string for that version.

Reply:
*Actually*, this is even simpler and seems to work:
> `rtb_buffs_will_lose < buff.loaded_dice.up` | buff.loaded_dice.up & rtb_buffs_will_lose < 3 & ( ! rtb_buffs_will_lose_buff.broadside | buff.broadside.remains < 11 ) & ( ! rtb_buffs_will_lose_buff.ruthless_precision | buff.ruthless_precision.remains < 11 ) & ( ! rtb_buffs_will_lose_buff.true_bearing | buff.true_bearing.remains < 11 )

That is, just removing the parentheses allows the addon to invoke `safenum` here.

Reply:
Which is easier / closer to simc for doing auto-translation in the future?